### ISSUE TITLE
TSServe reloading when extensions is enabled 

### ISSUE THREAD

**Issue by antoniel** [denoland/vscode_deno#1039]
Type: <b>Bug</b>

Just by open my vscode and try to have any intelisnse from typescript server it creates a loop by starting and stopping tsserver

Extension version: 3.32.0
VS Code version: Code - Insiders 1.86.0-insider (Universal) (1091f68d834974d78cd4f9769034b4ae4581ff2a, 2024-01-24T05:37:21.009Z)
OS version: Darwin arm64 23.1.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|4, 5, 4|
|Memory (System)|16.00GB (2.76GB free)|
|Process Argv|--crash-reporter-id 1be577f0-d069-49ad-ad6f-17be02adeecb|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscod805:30301674
vsaa593:30376534
py29gd2263:30784851
c4g48928:30535728
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
b5d27386:30943795
fegfb526:30945144
bg6jg535:30946824

```

</details>

**Comment by Joeao** [comment]
Also experienced this

Extension version: 3.32.0
VSCode Version: 1.85.2 (Universal) (8b3775030ed1a69b13e4f4c628c612102e30a681, 2024-01-18T06:40:32.531Z)
OS: MacOS 14.1, Macbook Air M2

**Comment by Mqxx** [comment]
Same problem here.

- Extension:
```
Version 3.31.0 works fine
Version 3.32.0 and above crashes TSServer
```

- VS-Code:
```
Version: 1.85.1 (system setup)
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:49:37.021Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19045
```

- Error log:
```log
2024-01-25 07:52:28.916 [info] Starting TS Server undefined
2024-01-25 07:52:28.916 [info] Using tsserver from: c:\Users\...\.vscode\extensions\ms-vscode.vscode-typescript-next-5.4.20240124\node_modules\typescript\lib\tsserver.js undefined
2024-01-25 07:52:28.916 [info] <syntax> Forking...
2024-01-25 07:52:28.916 [info] <syntax> Starting...
2024-01-25 07:52:28.916 [info] <semantic> Forking...
2024-01-25 07:52:28.916 [info] <semantic> Starting...
2024-01-25 07:52:38.025 [info] Killing TS Server undefined
2024-01-25 07:52:38.094 [error] TSServer exited. Code: null. Signal: SIGTERM
```

My current solution is to just downgrade to version `3.31.0`.
![image](https://github.com/denoland/vscode_deno/assets/62719703/167af228-76da-4547-bf0c-8af7e657d89e)

![image](https://github.com/denoland/vscode_deno/assets/62719703/338d5ad8-a5ae-4d98-bbac-d2183e6d579f)

**Comment by nayeemrmn** [comment]
Fixed in 3.33.2
